eng/create-announcement-draft.sh

```diff
@@ -6,23 +6,27 @@ print-help ()
   echo "Uses a template to draft a release announcement and outputs it into a file."
   echo
   echo "Options:"
-  echo "--template            The base template to use for the announcement"
-  echo "--channel             Release channel, e.g. '8.0'"
+  echo "--template=FILE       The base template to use for the announcement"
```

Comment:
Do things actually parse correctly if you use the "equals" syntax (`--template=FILE`) instead of using a space (`--template FILE`)?

Reply:
Fair point, I will put spaces in the man